sql/setup.sql

```diff
@@ -1,2 +1,23 @@
 -- Use this file to define your SQL tables
 -- The SQL in this file will be executed when you run `npm run setup-db`
+DROP table if exists cats;
+
+CREATE table cats (
+    id BIGINT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR NOT NULL,
+    type VARCHAR NOT NULL,
+    url VARCHAR,
+    year INT NOT NULL,
+    lives INT NOT NULL,
+    is_sidekick BOOLEAN NOT NULL
+);
+
+INSERT INTO cats (id, name, type, url, year, lives, is_sidekick) VALUES
```

Comment:
when inserting manually, you do not need to include ID.  id is automatically generated for you.